test/TimedAssetLockContract_test.py

```diff
@@ -59,7 +95,8 @@ def test_on_setup(self, app_id, test_config, client):
 
         public_key = test_config['wallet1_publicKey']
         creator_mnemonic = test_config['wallet1_Mnemonic']
-        asset_id = test_config['asset_id']
+        #asset_id = test_config['asset_id']

```

Comment:
in the setup code we can probably add in code that creates a new asset and then we just read in that asset here instead of from the testConfig